ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -143,32 +139,40 @@ export const HeaderKebab = ({ reportLinkURL, isConnected }) => {
             {_("Report Issue")}
         </DropdownItem>,
     ];
+
     return (
         <>
             <Dropdown
-              position={DropdownPosition.right}
+              isOpen={isOpen}
               onSelect={onSelect}
-              toggle={
-                  <KebabToggle
+              popperProps={{ position: "right" }}
+              position="right"
+              toggle={toggleRef =>
+                  <MenuToggle
+                    className="pf-m-align-right"
                     id="toggle-kebab"
-                    onToggle={(_event, isOpen) => onToggle(isOpen)}
-                  />
-              }
-              isOpen={isOpen}
-              isPlain
-              dropdownItems={dropdownItems}
-            />
+                    isExpanded={isOpen}
+                    onClick={onToggle}
+                    ref={toggleRef}
+                    variant="plain">
+                      <EllipsisVIcon />
+                  </MenuToggle>}
+              shouldFocusToggleOnSelect>
+                <DropdownList>
+                    {dropdownItems}
+                </DropdownList>
+            </Dropdown>
             {isAboutModalOpen &&
-            <AnacondaAboutModal
-              isModalOpen={isAboutModalOpen}
-              setIsAboutModalOpen={setIsAboutModalOpen}
-            />}
+                <AnacondaAboutModal
+                  isModalOpen={isAboutModalOpen}
+                  setIsAboutModalOpen={setIsAboutModalOpen}
+                />}
             {isReportIssueOpen &&
-            <UserIssue
-              reportLinkURL={reportLinkURL}
-              setIsReportIssueOpen={setIsReportIssueOpen}
-              isConnected={isConnected}
-            />}
+                <UserIssue
+                  isConnected={isConnected}
```

Comment:
This change is fine as afix, but unrelated to the commit. Please avoid introducing code improvements within random commits.